.gitignore

```diff
@@ -6,3 +6,4 @@
 build
 source/locale/pot/.doctrees
 /source/locale/en_US
+sphinx-mixxx/sphinx_mixxx.egg-info/
```

Comment:
Never seen this file, where does it come from?
I suspect this appears due to some custom cfg? If so, it belongs into your user ignore list.

Reply:
It gets spit out by Python sometimes when you do `make html`. It's unclear to me why as I thought eggs were long deprecated, but it's sometimes appearing for me and sometimes not when building in a local venv.

Reply:
Actually, I can't get this to happen now. Maybe I manually ran sphinx the first time or something? Unclear. Egg files definitely shouldn't be committed if they *are* generated somehow, but it doesn't appear to be a regular problem. I can drop the commit if you prefer.

Reply:
Hmm, the ignore section is `# Generated during a regular build` and I did a lot of 'regular builds' (html, pdf has been a while) so I prefer to remove it.
Let us know when it's bee created again.

Reply:
Dropped the commit and squashed everything else for good measure. Thanks for the feedback!

Reply:
I realized right after dropping this that I know exactly how to regenerate it; if you are installing sphinx itself (and other requirements) in a virtual env, eg. via `pip install requirements.txt`, this file will always be generated. If you're using a system wide installation of the theme and sphinx this isn't necessary, but for systems like mine that don't have one or more of these dependencies packaged (Arch is missing the SVG converter stuff) it's necessary to install them yourself using pip. Happy to leave it as is, or pull the commit back in. Whatever works best for you.

Reply:
I can confirm this: egg has been created by
`pip3 install -r requirements.txt --upgrade`

@Holzhaus what's your opinion on this?
Add to Mixxx' or personal ignore list?

Reply:
In the mean time, is there anything else needed for this specific PR since this commit is unrelated and can be followed up on afterwards?

Reply:
This PR is green now so I created a follow up for the .gitignore issue in #736 so that it doesn't get forgotten about and we don't have to block this PR on an unrelated change.